test/src/sdk-harness/test_projects/storage_map/try_insert.rs

```diff
@@ -0,0 +1,64 @@
+use super::*;
+
+#[macro_export]
+macro_rules! generate_try_insert_tests {
+    ($input_string:expr, $key:expr, $value1:expr, $value2:expr) => {
+        paste::paste! {
```

Comment:
I've spent about 2 minutes looking at the paste docs. What functionality does it provide that macros do not?

Reply:
You cannot make up function names using regular macros